server/protocols/webadmin/webadmin-cassandra/src/test/java/org/apache/james/webadmin/routes/CassandraMigrationRoutesTest.java

```diff
@@ -232,6 +232,18 @@ public void postShouldNotUpdateVersionWhenCurrentVersionIsNewerThan() {
         verifyNoMoreInteractions(schemaVersionDAO);
     }
 
+    @Test
+    public void postShouldPositionLocationHeader() {
+        when(schemaVersionDAO.getCurrentSchemaVersion()).thenReturn(Mono.just(Optional.of(CURRENT_VERSION)));
+
+        given()
+            .body(String.valueOf(OLDER_VERSION.getValue()))
+        .when()
+            .post("/upgrade")
+            .then()
+            .header("Location", notNullValue());
```

Comment:
what is the purpose of this header?

Reply:
It indicates that a resource related to this call had been created on another endpoint